tools/deb/istio-start.sh

```diff
@@ -39,8 +39,11 @@ ISTIO_CFG=${ISTIO_CFG:-/var/lib/istio}
 NS=${ISTIO_NAMESPACE:-default}
 SVC=${ISTIO_SERVICE:-rawvm}
 ISTIO_SYSTEM_NAMESPACE=${ISTIO_SYSTEM_NAMESPACE:-istio-system}
-ISTIO_PILOT_PORT=${ISTIO_PILOT_PORT:-15003}
-CONTROL_PLANE_AUTH_POLICY=${CONTROL_PLANE_AUTH_POLICY:-MUTUAL_TLS}
+
+# The default matches the default istio.yaml - use sidecar.env to override this if you
+# enable auth. This requires node-agent to be running.
+ISTIO_PILOT_PORT=${ISTIO_PILOT_PORT:-8080}
+CONTROL_PLANE_AUTH_POLICY=${CONTROL_PLANE_AUTH_POLICY:-NONE}
```

Comment:
it's quite longish but this should probably have been ISTIO_CONTROL_PLANE_AUTH when first added

Reply:
Just used what the script was using - agree, we can tweak it later.

Reply:
Updated to ISTIO_CP_AUTH